css/documents.css

```diff
@@ -0,0 +1,153 @@
+.tna-breadcrumb .breadcrumbs {
```

Comment:
@sdwilkes , yes this is a redundant css rule. I have updated this in the sass file now.